home-manager.nix

```diff
@@ -73,28 +73,29 @@ in
                 dirListToPath (tail (splitPath [ dir ]))
               else
                 dir;
-            targetDir = concatPaths [ persistentStoragePath dir ];
-            mountPoint = concatPaths [ config.home.homeDirectory mountDir ];
+            targetDir = escapeShellArg (concatPaths [ persistentStoragePath dir ]);
+            mountPoint = escapeShellArg (concatPaths [ config.home.homeDirectory mountDir ]);
             name = "bindMount-${sanitizeName targetDir}";
             startScript = pkgs.writeShellScript name ''
               set -eu
-              if ! ${pkgs.utillinux}/bin/mount | ${pkgs.gnugrep}/bin/grep -E '${mountPoint}( |/)'; then
-                  ${pkgs.bindfs}/bin/bindfs -f --no-allow-other "${targetDir}" "${mountPoint}"
+              if ! ${pkgs.utillinux}/bin/mount | ${pkgs.gnugrep}/bin/grep -F ${mountPoint}' ' \
+                 && ! ${pkgs.utillinux}/bin/mount | ${pkgs.gnugrep}/bin/grep -F ${mountPoint}/; then
+                  ${pkgs.bindfs}/bin/bindfs -f --no-allow-other ${targetDir} ${mountPoint}
```

Comment:
I feel like it'd be better to bind these binaries to variables in Nix, like `mount = "${pkgs.utillinux}/bin/mount";` and then using them as `${mount}` instead of having the long path everywhere.

Reply:
Yeah, that sounds a lot cleaner. :)